labgrid/driver/dockerdriver.py

```diff
@@ -0,0 +1,106 @@
+import attr
+import docker
```

Comment:
Please import this where needed so labgrid also works without the docker package installed (and also keep the startup time lower).

Reply:
See comments from trxanga